### ISSUE TITLE
New logo?

### ISSUE THREAD

**Issue by alex-s-gardner** [rafaqz/DimensionalData.jl#778]
For whatever reason I've never liked the DD logo. I find it very cluttered and hard to identify from a branding perspective. Given that DD is going to play a foundational role in building out the raster echo system I thought I would propose a new logo that could be more easily recognized.  Take it or leave it. Happy to iterate. 

![DimensionalDataLogo](https://github.com/user-attachments/assets/0d5c1d02-8e17-4539-811b-73793dd5d4d5)

**Comment by lazarusA** [comment]
fair. But, the original idea was to represent some DD primitives, and not just do cubes.

Points, `.    .`
Intervals, `. ---- .`
and the mix: two intervals or more, creating squares of volumes of data. I agree, a simplification might be needed. 

Also, for whatever reason, we are not indexing any `image` files, so the logo is autogenerated when deploying:

Here is the code to work with: https://github.com/rafaqz/DimensionalData.jl/blob/main/docs/logo.jl

**Comment by rafaqz** [comment]
Yeah, we didn't put that much time into it, it was a small part of the vitepress docs and repl bling overhaul that we did. I'm open to better logos. I much prefer the DD colorscheme to the julia colors, but your image does get the message accross.

We can also embed an image in a comment somewhere if we need to.

**Comment by alex-s-gardner** [comment]
I think the intention of the current graphic is opaque unless you helped to develop the package. I didn't realize that it was an attempt to show primitives ... even now I find that hard to tease from the graphic. 

Julia colors might not be my favorite but looking at the proposed graphic the user immediately knows that it's some kind of julia package for dealing with array datasets (I know.. it's more than that)

it's also in family with YAXArray which I only looked at after I created the above:

![image](https://github.com/user-attachments/assets/47d49601-b5c0-40a5-a0e7-df573674d073)

**Comment by alex-s-gardner** [comment]
Or what about something like this? It's a crude mockup... my idea is that it's a transparent "rubik's cube" with a column, interval and point selected.

<img width="762" alt="Screenshot 2024-08-25 at 11 33 14 AM" src="https://github.com/user-attachments/assets/ca879940-83e9-4a0d-ad71-7a3ea510f747">

**Comment by rafaqz** [comment]
The first one is cleare

**Comment by alex-s-gardner** [comment]
Including here for future PR
![DimensionalDataLogo](https://github.com/user-attachments/assets/781ae61d-3316-4c69-a128-6626ea8a3a37)

**Comment by rafaqz** [comment]
![2024-08-26-220511_1304x451](https://github.com/user-attachments/assets/238134e0-5112-4958-aa17-2d7085946a0f)

**Comment by rafaqz** [comment]
Those colors my god. On the other hand it is more recognizable.

I think we need something better to justify changing, maybe a simplification of what we have. Idk. I don't have any spare time to think about it personally for the next two months at least

**Comment by alex-s-gardner** [comment]
![DimensionalDataLogo_v2](https://github.com/user-attachments/assets/84c99edd-2971-483e-8d09-589a6e0c63ba)

**Comment by alex-s-gardner** [comment]
<img width="872" alt="Screenshot 2024-08-28 at 10 11 15 AM" src="https://github.com/user-attachments/assets/b5540c6f-9ca0-4e1d-9728-a75895d74c2c">

**Comment by alex-s-gardner** [comment]
![DimensionalDataLogo_v3](https://github.com/user-attachments/assets/fae06b7a-ed02-4acb-8d0d-de0121e7b6f6)

**Comment by alex-s-gardner** [comment]
<img width="859" alt="Screenshot 2024-08-28 at 10 17 43 AM" src="https://github.com/user-attachments/assets/62cb6ce0-56c4-41fd-846a-e5dea16f2a77">

**Comment by alex-s-gardner** [comment]
For easy comparison to what we have now:
<img width="868" alt="Screenshot 2024-08-28 at 10 21 01 AM" src="https://github.com/user-attachments/assets/2ffa262a-30e7-4680-933d-a2ce5625a815">

**Comment by lazarusA** [comment]
that's better. However, I would still argue that DD is not just for xarray-like data structures, hence having some representation for the other primitives (like points and intervals) should be visible somehow. Maybe, by replacing some of the white cubes.

**Comment by alex-s-gardner** [comment]
Can an interval and a point not also be represented by a voxel? I'm having trouble distinguishing.

**Comment by asinghvi17** [comment]
I like the design but not a huge fan of the colors 😅 - are they open to change?

**Comment by alex-s-gardner** [comment]
Color and design option encouraged.

**Comment by asinghvi17** [comment]
Could you post the script you used to generate it?  (assuming this was with Makie.voxels but may be wrong)

**Comment by alex-s-gardner** [comment]
Well this is embarrassing: 
[DimensionalDataLogo.pptx](https://github.com/user-attachments/files/16790195/DimensionalDataLogo.pptx)

**Comment by rafaqz** [comment]
Probably anything we do should use the DD REPL colours. They're listed in Dimensions/show.jl

**Comment by asinghvi17** [comment]
These seem a bit inscrutable - are they terminal codes?

https://github.com/rafaqz/DimensionalData.jl/blob/5caae5792cc969bf97756fae8eddc8420e2618e6/src/Dimensions/show.jl#L11-L13

**Comment by asinghvi17** [comment]
```julia
using GLMakie, Colors
dd_colors = [colorant"#ff875f", colorant"#0087d7", colorant"#5fd7ff", colorant"#ff5f87", colorant"#b2b2b2"]
voxels(rand(UInt8.(1:5), 5, 5, 5); color = Colors.alphacolor.(dd_colors, 0.3), gap = 0.3, axis = (; type = LScene, show_axis=false))
```
![hello](https://github.com/user-attachments/assets/84fbe20f-5d06-42c0-9069-8d82fb83ea70)

Unfortunately the rendering is not 100%

**Comment by alex-s-gardner** [comment]
Ohhh… I like it !

**Comment by asinghvi17** [comment]
You can control transparency per color but not per voxel.  There's a limit of 256 colors though so you could have multiple colors with the same RGB but different A.

**Comment by asinghvi17** [comment]
You can control transparency per color but not per voxel.  There's a limit of 256 colors though so you could have multiple colors with the same RGB but different A.

**Comment by alex-s-gardner** [comment]
OK, upping our game with @asinghvi17 guidence:

Gif version:

![dd_logo_v5](https://github.com/user-attachments/assets/8ba46976-849f-47a1-a62e-dd4e46a356a0)

Static version:

![dd_logo_v5](https://github.com/user-attachments/assets/1ecf9749-25b5-4ba7-9968-5af4b1445ff3)

**Comment by alex-s-gardner** [comment]
Or the "DD" in morse code.. probably my favorite:

GIF
![dd_morse](https://github.com/user-attachments/assets/85ad2059-bb69-462f-9987-283bd9c729ad)

and Static:
![dd_morse](https://github.com/user-attachments/assets/bd47b26f-3fea-4b15-bef3-34cd30f9aa9f)

**Comment by lazarusA** [comment]
now, that breaking its out! should we revisit this? maybe? However, nowadays I don't even like the current logo 😓 , and the options above are not convincing, maybe having sets of 3, so that we can easily discard them (comparing side-by-side) could help.

**Comment by alex-s-gardner** [comment]
I'd love to see a new logo, I'm not a fan of what we have. It tries to convey way too much in a logo. @lazarusA I'm sure you have some great ideas and I'm happy to iterate. Right now my preference is DD in morse code.. but I'm sure we can do better:

![Image](https://github.com/user-attachments/assets/05971ca4-8918-4c68-a7ba-14b1498d9cf6)

**Comment by alex-s-gardner** [comment]
and here is Gemini's take:

<img width="1024" height="559" alt="Image" src="https://github.com/user-attachments/assets/9a521bd9-e460-46ba-bbd7-2f550018f786" />
